shared/transforms/relabelids.py

```diff
@@ -1,11 +1,22 @@
-#!/usr/bin/python2
+#!/usr/bin/python
 
 import re
 import sys
 import os
-import idtranslate
 import lxml.etree as ET
 
+try:
+    from configparser import SafeConfigParser
+except ImportError:
+    # for python2
+    from ConfigParser import SafeConfigParser
+
+# Put shared python modules in path
+sys.path.insert(0, os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
```

Comment:
Yeah, this double dirname stuff was annoying, I tried to find something more elegant but I don't think there is anything better in the python API. I hope to change it once again with more refactoring once I straighten the utils, transforms, modules directories so that each only contains what it should contain - XSLTs in transforms, scripts in utils and modules in modules. It's better after this PR but still all over the place..

What makes this more annoying is that it's a magic snippet that you can't put in one place :-) You need it to include any python modules so you can't place it in the shared modules. I hope I will be able to use python relative imports once I move things around a little bit but right now that's not possible either.